src/components/Header/subcomponents/Cart.tsx

```diff
@@ -0,0 +1,69 @@
+'use client'
+
+import React, { useContext } from 'react'
+import styles from './Cart.module.sass'
+import CartItem from '@/components/shared/CartItem'
+import RedirectButton from '@/components/shared/RedirectButton'
+import { CartContext } from '@/context/CartContext'
+
+const Cart = ({
+  cartRef,
+  handleCartClick,
+}: {
+  cartRef: React.RefObject<HTMLDivElement>
+  handleCartClick: () => void
+}) => {
+  const { cart, cartTotalPrice, deleteFromCart } = useContext(CartContext)
+
+  const handleClearCart = () => {
+    cart.forEach((item) => deleteFromCart(item.title))
```

Comment:
is this going to remove all the items in the cart?

Reply:
yes, added a comment to explain function